src/palace/manager/api/axis.py

```diff
@@ -662,11 +662,10 @@ def update_book(
             analytics=analytics,
         )
 
-        # NOTE: availability is bibliographic.circulation, so it's a
-        # little redundant to call availability.apply() -- it's taken
+        # NOTE: availability is bibliographic.circulation, so it is
+        # unnecessary to call availability.apply() -- it's taken
         # care of inside bibliographic.apply().
         bibliographic.apply(edition, self.collection, replace=policy, db=self._db)
-        availability.apply(self._db, self.collection, replace=policy)
```

Comment:
I'd suggest removing the `availability` parameter to this function and updating the functions doc comment. The parameter is unused after this change, and its misleading, since passing an availability to this function won't do anything with it, unless its already referenced by `bibliographic`.